src/main/java/com/minecolonies/commands/MinecoloniesCommand.java

```diff
@@ -1,6 +1,7 @@
 package com.minecolonies.commands;
 
 import com.google.common.collect.ImmutableMap;
+import net.minecraft.command.CommandBase;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.command.CommandBase'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)